Misc/NEWS.d/next/IDLE/2018-02-12-11-05-22.bpo-32826.IxNZrk.rst

```diff
@@ -0,0 +1,5 @@
+Add "encoding=utf-8" to open() in idle_test/test_help_about.
+
```

Comment:
I think these empty lines are not needed. Bug in blurb?

Reply:
Yes, they were not in the file as I wrote it.  Fixed with online editor.